src/wrappers/React.js

```diff
@@ -67,7 +68,9 @@ export default {
      * AFAIK, this is the only way to update children. It doesn't seem to be possible to watch
      * `$slots` or `$children`.
      */
-    this.reactComponentRef.setState({ children: this.$slots.default })
+    if (this.$slots.default) {
```

Comment:
Here, we should compare `$slots.default` to `undefined`, and also, in case `$slots.default === undefined`, we should set React component's `state.children` to `null`.

Reply:
👍

Reply:
why i console.log(this.$slots.default) in your code and see the result in chrome console, but it didn't render